src/server/game/Server/Packets/CalendarPackets.h

```diff
@@ -0,0 +1,576 @@
+
+            void Read() override;
+
+            uint64 ModeratorID = 0;
+            uint64 EventID = 0;
+            uint32 Flags = 0;
+        };
+
+        class CalendarCopyEvent final : public ClientPacket
+        {
+        public:
+            CalendarCopyEvent(WorldPacket&& packet) : ClientPacket(CMSG_CALENDAR_COPY_EVENT, std::move(packet)) { }
+
+            void Read() override;
+
+            uint64 ModeratorID = 0;
+            uint64 EventID = 0;
+            time_t Date = time_t(0);
+        };
+
+        class SCalendarEventInvite final : public ServerPacket
+        {
+        public:
+            SCalendarEventInvite() : ServerPacket(SMSG_CALENDAR_EVENT_INVITE, 43) { }
+
+            WorldPacket const* Write() override;
+
+            uint64 InviteID = 0;
+            time_t ResponseTime = time_t(0);
+            uint8 Level = 100;
+            ObjectGuid InviteGuid;
+            uint64 EventID = 0;
+            uint8 Type = 0;
+            bool ClearPending = false;
+            uint8 Status = 0;
+        };
+
+        struct CalendarSendCalendarInviteInfo
+        {
+            uint64 EventID = 0;
+            uint64 InviteID = 0;
+            ObjectGuid InviterGuid;
+            uint8 Status = 0;
+            uint8 Moderator = 0;
+            uint8 InviteType = 0;
+        };
+
+        struct CalendarSendCalendarHolidayInfo
+        {
+            int32 HolidayID = 0;
+            int32 Region = 0;
+            int32 Looping = 0;
+            int32 Priority = 0;
+            int32 FilterType = 0;
+            std::string TextureFilename;
+            int32 Date[26] = { };
+            int32 Duration[10] = { };
+            int32 CalendarFlags[10] = { };
+        };
+
+        struct CalendarSendCalendarRaidLockoutInfo
+        {
+            uint64 InstanceID = 0;
+            int32 MapID = 0;
+            uint32 DifficultyID = 0;
+            time_t ExpireTime = time_t(0);
+        };
+
+        struct CalendarSendCalendarRaidResetInfo
+        {
+            int32 MapID = 0;
+            uint32 Duration = 0;
+            int32 Offset = 0;
+        };
+
+        struct CalendarSendCalendarEventInfo
+        {
+            uint64 EventID = 0;
+            std::string EventName;
+            uint8 EventType = 0;
+            time_t Date = time_t(0);
+            uint32 Flags = 0;
+            int32 TextureID = 0;
+            ObjectGuid EventGuildID;
+            ObjectGuid OwnerGuid;
+        };
+
+        class CalendarSendCalendar final : public ServerPacket
+        {
+        public:
+            CalendarSendCalendar() : ServerPacket(SMSG_CALENDAR_SEND_CALENDAR, 338) { }
+
+            WorldPacket const* Write() override;
+
+            time_t RaidOrigin = time_t(0);
+            time_t ServerTime = time_t(0);
+            time_t ServerNow = time_t(0);
+            std::vector<CalendarSendCalendarHolidayInfo> Holidays;
```

Comment:
Why is this here? This packet no longer contains holiday info (it was moved to SMSG_DB_REPLY as Holidays is now a db2 storage)